cmd/helm/status.go

```diff
@@ -39,7 +39,7 @@ The status consists of:
 - last deployment time
 - k8s namespace in which the release lives
 - state of the release (can be: unknown, deployed, uninstalled, superseded, failed, uninstalling, pending-install, pending-upgrade or pending-rollback)
-- list of resources that this release consists of, sorted by kind
```

Comment:
Add this back in as it should still be output and there is/are bug(s) currently opened to fix it.